.github/workflows/scripts/install-build-deps.sh

```diff
@@ -38,6 +38,8 @@ for i in {1..5}
 do
   sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y && sudo apt-get -y update | tee log${i}.txt
   sudo apt-get install ninja-build bison flex libtbb-dev libstdc++6 "${CROSS_LIBS[@]}" | tee -a log${i}.txt
+  sudo apt install python3-pip python3-dev
+  pip3 install nanobind numpy
```

Comment:
Just a comment, no need to change anything. I think we should eventually add requirements.txt to the repo with all python deps. The security requirement will be to add them with pinned versions (this can be a little challenging due to wide range of platforms we support in testing).